src/api/graphql/schema.ts

```diff
@@ -0,0 +1,89 @@
+import { YogaInitialContext, createSchema } from 'graphql-yoga';
+import { EventLogFromDb, IndexingStatus, PersistenceObject } from '../../types';
+import { FilterOperators, FilterTypes, SortClosure, WhereClosure } from '../../database/filters';
+import { parseEventArgs } from '../../utils/parseEventArgs';
+
+export type IndexerContext = YogaInitialContext & { db: PersistenceObject; chainId: number };
+
+const gpqSchema = `
+    type EventArg {
+        from: String!
+        to: String!
+        value: String!
+    }
```

Comment:
yep you are right. just did it for graphql testing. now it will be a usual scalar object, but without knowing schema upfront we won't be able to filter any properties under args.